.github/workflows/r-check.yaml

```diff
@@ -59,6 +59,7 @@ jobs:
           r-version: ${{ matrix.config.r }}
           http-user-agent: ${{ matrix.config.http-user-agent }}
           use-public-rspm: true
+          install-quarto: false
```

Comment:
GH actions seems to have been broken by https://github.com/r-lib/actions/blame/d1a6cbabea8dd2f137598ba2a70ae37ac82d7941/setup-r-dependencies/action.yaml#L215

which tries setting up quarto by default. I'm not sure how else to disable this

Reply:
... or alternatively to allow quarto to setup https://github.com/apache/arrow-nanoarrow/actions/runs/10220526007/job/28281045091?pr=555#step:1:39

Reply:
hm not sure if we use quarto in any of our jobs, if we do I can get it allow-listed :+1: